.eslintrc.yml

```diff
@@ -21,6 +21,8 @@ settings:
     onlyFilesWithFlowAnnotation: true
 globals:
   __DEV__: true
+  ReactClass: true
+  ReactElement: true
```

Comment:
This is crazy train, I [can't find](https://github.com/facebook/flow/blame/master/lib/react.js) anything about `ReactElement` here. AFAIR I had issues w/ it and bunch of examples were using `React$Element`—it is in the defs, but it wasn't work either. Then I found [this post](http://technologyadvice.github.io/eradicate-runtime-errors-in-react-with-flow/), where stated that `ReactElement` is removed from public API and `React.Element` should be used instead. The funny thing is that it worked and still works, but: there is no `Element` property on `React` object lol.